cmd/eskip/media_defaults.go

```diff
@@ -0,0 +1,45 @@
+package main
```

Comment:
I will rename it... I don't think we should be afraid of creating more files... it keeps the code more structured... also important is that it keeps your tests structured...